build.gradle

```diff
@@ -62,6 +67,12 @@ allprojects {
         exclude group: 'ch.qos.logback', module: 'logback-core'
         exclude group: 'org.apache.logging.log4j', module: 'log4j-to-slf4j'
         exclude group: 'org.slf4j', module: 'slf4j-log4j12'
+
+        // Force contribs to use local project module
+        resolutionStrategy.dependencySubstitution {
```

Comment:
Force all `conductor-common` and `conductor-core` dependency to refer to the local project's module